index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week1</title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+    <header class = "header1">
+        <nav>
+            <ul>
+                <li><a href=""> <img src="images/swiggylogo.png" alt = "swiggylogo" class = "swiggylogo"> </a></li>
+                <li> <a href="" class="setloc">Set location</a> </li>
+                <li class="chooseloc"> <a href="">Click to Choose Location</a> </li>
+            </ul>
+            <ul>
+                <!-- <li><a href=""></a></li> -->
+                <li><a href="">Help</a></li>
+                <li><a href="">Sign In</a></li>
+                <li><a href="">Cart</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class = "header2">
+        <nav>
+            <ul>
+                <li><a href="">Home</a> <small>/</small></li>
+                <li><a href="">Banglore</a> <small>/</small></li>
+                <li><a href="">Ashok Nagar</a> <small>/</small></li>
+                <li><a href="">Meghana Foods</a></li>
+            </ul>
+        </nav>
+    </section>
+
+    <section class="box1">
+        <img src="images/box1_image.jpeg" alt = "Food image">
+        <h1><strong>Punjabi Momos</strong></h1>
+    </section>
+
+    <main>
+        <div class="categories">
+            <h4>Recommended</h4>
```

Comment:
Headings should follow a structure, this cannot be an `h4`

Also, as discussed make this a category and not a heading